### ISSUE TITLE
Annoying auto-complete behavior

### ISSUE THREAD

**Issue by disarray2077** [beefytech/Beef#1951]
Auto-complete has this annoying behavior that triggers an auto-complete while you're trying to write parameters in a lambda expression. This doesn't happen if you make it explicit that you want to create a delegate (e.g., by prepending scope or new), but in the case below, I want to make a method reference.

https://github.com/beefytech/Beef/assets/86157825/439020c0-f4da-4d92-8e1b-e9bfcc2edd6e

**Comment by bfiete** [comment]
Yeah, that is annoying. I don't see a really good solution, though. At that point it looks like you're performing a cast... 

I would say that you could disambiguate by providing a capture specifier like `[](i) =>` instead of just the `(i) =>` but capture specifiers don't currently parse on method references.
